.github/workflows/release.yml

```
@@ -15,10 +19,28 @@ jobs:
         with:
           distribution: 'temurin'
           java-version: 11
+          cache: 'gradle'
       - name: Setup Gradle
         uses: gradle/actions/setup-gradle@v4
-      - name: Publish with Gradle
-        run: ./gradlew publish
+      - name: Update version in gradle.properties
+        run: |
+          sed -i 's/^version=.*$/version=$VERSION/' gradle.properties
```

Maybe I'm missing something, but I think you have to use double quotes for variable expansion to work. Unless sed does some magic variable replacement as well?

```suggestion
          sed -i "s/^version=.*$/version=$VERSION/" gradle.properties
```

---

Ah yes, good point. I tried it with a fixed version on the console, but forgot to change up the quotes